Add MV2 timeline blog post & known issues doc (#3861)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/whatsnew/index.md

title: What's new in Chrome extensions
description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
date: 2021-02-25
updated: 2022-09-28

# Note: disabling the linter for duplicate headings because this isn't hierarchical and it needs
# smaller font headings.
---
<!--lint disable no-duplicate-headings-->
<!--lint disable first-heading-level-->

Check this page often to learn about changes to the Chrome extensions platform, its documentation,
and related policy or other changes.

### Blog post: More details on the transition to Manifest V3 {: #blog-mv3-transition-details }

September 28, 2022

> [This blog post](/blog/more-mv2-transition/) provides clarifications on the Manifest V2 deprecation timeline. The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.

Reviewer: ```suggestion
We have [clarified the Manifest V2 deprecation timeline](/blog/more-mv2-transition/). The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
```

### Docs update: Known issues when migrating to Manifest V3 {: #known-issues }

September 28, 2022

> We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/).

Author: ATM this feels a bit terse. Is there anything else that we could say? I was considering something along the lines of "Our goal with this page is to help developers better understand the current state of the platform and what features they can target as they prepare for the future", but I'm on the fence about it.

Reviewer: SGTM

> We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/).

Author: ```suggestion
We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/). Our goal with this page is to help developers better understand the current state of the platform and what features they can target as they prepare for the future.
```

> [This new article](/docs/extensions/mv3/known-issues/) provides an overview of major features currently in development and open bugs.

Reviewer: ```suggestion
We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/).
```

> [This new article](/docs/extensions/mv3/intro/) provides an overview of major features currently in development and open bugs.

Reviewer: This isn't a new article. Is this the right link?

Author: Oops, wrong page! I meant to link to the known issues page. 

```suggestion
[This new article](/docs/extensions/mv3/known-issues/) provides an overview of major features currently in development and open bugs.
```
